tensorflow/python/keras/benchmarks/BUILD

```diff
@@ -91,3 +92,16 @@ py_library(
         "//third_party/py/numpy",
     ],
 )
+
+py_library(
+    name = "perfzero_benchmark_lib",
+    srcs = [
+        "perfzero_benchmark.py",
+    ],
+    deps = [
+        "@absl_py//absl/flags",
+        "@absl_py//absl/logging",
+        "@absl_py//absl/testing:flagsaver",
```

Comment:
`@` looks not right.

Reply:
> `@` looks not right.

I checked with current tensorflow repo and use same format. You can find from [here](https://github.com/tensorflow/tensorflow/search?q=%40absl_py%2F%2Fabsl%2Flogging&unscoped_q=%40absl_py%2F%2Fabsl%2Flogging).

Reply:
emm... that's strange. Can it work correctly if you run the benchmark locally with bazel?

Reply:
> emm... that's strange. Can it work correctly if you run the benchmark locally with bazel?

it worked. For the current failed checks, it doesn't relate to our code from log.

Reply:
All right. It's fine as long as it works. copybara should be able to handle it then.